fissa/extraction.py

```diff
@@ -133,9 +132,24 @@ def image2array(image):
             A 3D array containing the data, with dimensions corresponding to
             ``(frames, y_coordinate, x_coordinate)``.
         """
-        if isinstance(image, basestring):
-            return tifffile.imread(image)
-        return np.array(image)
+        if not isinstance(image, basestring):
+            return np.array(image)
```

Comment:
We could throw a TypeError() if the input is neither an array or basestring, perhaps even with the suggestion to write a new datahandling method. We ask for array-like inputs so it can also handle lists I guess, so perhaps we can have:

```suggestion
            if isinstance(image, (np.ndarray, list)):
                   return np.array(image)
            else:
                   raise TypeError('Wrong image input format: unfamiliar shape. Expected array_like or string.')
```

Although lists of things can be lots of things and probably not good to expect.

Reply:
I think it is better to do duck typing - anything which is array-like should be handled in this way. It could be a list of lists. It could be a tuple instead of a list. Or it could be a pandas.DataFrame. Or any arbitrary array-like data structure.

Either way, I've not altered this bit of the code (the behaviour w.r.t. non-strings is the same as it was before), so changing it is off-topic for this PR.

Incidentally, I think it would be better to use np.asarray instead of np.array, since np.array will make a copy of the memory if the input is already an ndarary and np.asarray doesn't make a copy.